caas/kubernetes/provider/statefulsets.go

```diff
@@ -32,33 +32,20 @@ func (k *kubernetesClient) configureStatefulSet(
 		return applicationConfigMapName(deploymentName, fileSetName)
 	}
 
-	existingStatefulSet, err := k.getStatefulSet(deploymentName)
-	if err != nil && !errors.IsNotFound(err) {
+	randPrefix, err := k.getStorageUniqPrefix(func() (annotationGetter, error) {
```

Comment:
this isn't really a randPreix anymore.
maybe randPrefixFunc?

Reply:
It's still a randPrefix, but just changed to `storageUniqueID` to make it more explicit.